test/WebJobs.Script.Tests.Integration/Management/MeshServiceClientTests.cs

```diff
@@ -55,7 +55,7 @@ private static bool IsPublishExecutionStatusRequest(HttpRequestMessage request,
         {
             var formData = request.Content.ReadAsFormDataAsync().Result;
             if (string.Equals(MeshInitUri, request.RequestUri.AbsoluteUri) &&
-                string.Equals("add-fes", formData["operation"]))
+                string.Equals(MeshServiceClient.AddFES, formData["operation"]))
```

Comment:
I believe your Deserialize call is also exposed to this issue. Should probably specify explicit settings on that call as well.